debian/changelog

```diff
@@ -0,0 +1,5 @@
+motor (3.5.1-1) unstable; urgency=medium
```

Comment:
i'm not a fan of having distro specific additions inside my projects source trees. these can be added into the external packaging projects.

Reply:
oh

Reply:
like a submodule?

Reply:
no, add it to the package description/build in the ppa like the other debian packages do. usually they ship debian/ contents in a separate tarball, afaik.

Reply:
Kind of. The different tarballs are generated when the package is built

Reply:
They get uploaded to the Debian build servers where the package is compiled for different architectures and stuff

Reply:
The Debian folder has all of the metadata and configuration files needed for creation of source and binary packages. That folder needs to stay.

Reply:
as a submodule, if someone cloning the git repository doesn't want or need the debian stuff, they can clone it non-recursively

Reply:
there is probably less than 1% of packages in debian that have the debian directory in the upstream sources. you can literally lookup the first package that comes to your mind on the debian package repositories, check the official source tarball and see that debian/ is not inside.

Reply:
here is an example. https://packages.debian.org/stretch/tmux

Reply:
Those are generated from the source. They get posted there because that’s what gets uploaded to Debian. Yes there are separate tarballs for orig and debian but those are uploaded along with a release.

Reply:
The official source tarball doesn’t have it in there because it is separate from the source. The Debian directory is there in this case because of how I have the deb helper integration in motor set up

Reply:
The orig tarball still doesn’t have the Debian directory when generated from motor

Reply:
I also don’t want to have to configure the Debian directory every time I package motor. I usually do some things with moving it out, generating the orig tarball, deleting the Debian folder generated by dh_make, and replacing it with the folder I copied out earlier and changing the changelog and stuff, then build the source/binary package which generates the Debian tarball with the new folder in the process. It is a lot easier to do that then generate a new one every time and possibly make a mistake when trying to recreate the different files.

Reply:
Motor automates that process and therefore requires the Debian directory to be in that spot

Reply:
And btw you had a spec file in there that is definitely distro specific to rpm

Reply:
I moved the debian folder to my local machine only so that the upstream source doesn't have the folder